lib/database.js

```diff
@@ -45,10 +55,11 @@ function Database(filenameGiven, options) {
 	// Load the native addon
 	let addon;
 	if (nativeBinding == null) {
-		addon = DEFAULT_ADDON || (DEFAULT_ADDON = require('bindings')('better_sqlite3.node'));
+		addon = DEFAULT_ADDON || (DEFAULT_ADDON = bindings('better_sqlite3.node'));
```

Comment:
Previously, the `binding` function was lazy-loaded but now itis  always loaded. This will hurt cold start performance.